src/pages/billing/List/List.ts

```diff
@@ -139,9 +141,9 @@ export const List = ({ queryParams: { selectedName } }) => {
   const creatingAzureBillingProject = !selectedName && creatingBillingProject === cloudProviders.azure
 
   return h(FooterWrapper, { fixedHeight: true }, [
-    h(TopBar, { title: 'Billing' }, [
+    h(TopBar, { title: 'Billing', href: Nav.getLink('billing') }, [
```

Comment:
Before if you clicked on "Billing" (beside the Terra Logo) you would get kicked back to root. Now it behaves like it does within the workspaces views.

<img width="530" alt="image" src="https://user-images.githubusercontent.com/484484/225119769-220fed8a-d133-49ce-87ac-75ba8a51d670.png">